app/services/article_api_index_service.rb

```diff
@@ -121,6 +126,20 @@ def collection_articles(collection_id)
       .per(per_page || DEFAULT_PER_PAGE)
   end
 
+  def sorted_articles(sort)
+    # This could be expanded to allow additional sorting options
+    if sort == "desc"
+      Article.published
+        .where(featured: true)
+        .includes(:user, :organization)
+        .order(published_at: :desc)
+        .page(page)
+        .per(per_page || DEFAULT_PER_PAGE)
```

Comment:
non blocking: as this is essentially `base_articles`, you could recycle it with something like `base_articles.reorder(published_at: :desc)`

Reply:
I believe that would behave oddly because of the way we've written pagination on these endpoints. Your comment also made me realize that I had included a `where` clause that didn't need to be on this query :sweat_smile: Thanks!

Reply:
Cool! BTW you don't have to force push unless you're rebasing for specific reasons, we squash the changes when we merge, it's also a bit easier to follow the feedback cycle as it doesn't rewrite the history ☺️

Reply:
Yeah, I rebase and force push habitually :grimacing: 

In this case, I needed to get a fix from master to make CI pass. :smile: